src/include/distributed/commands.h

```diff
@@ -61,7 +61,23 @@ typedef enum ExtractForeignKeyConstraintsMode
 	INCLUDE_REFERENCED_CONSTRAINTS = 1 << 1,
 
 	/* exclude the self-referencing foreign keys */
-	EXCLUDE_SELF_REFERENCES = 1 << 2
+	EXCLUDE_SELF_REFERENCES = 1 << 2,
+
```

Comment:
still worth to add a comment here like this:

/* any combination of below 4 flags is supported */